### DISCUSSION TITLE
MikoPBX 2022.2.102

### DISCUSSION THREAD

**Discussion by jorikfon** [mikopbx/Core#377] [General ( Обо всём )]
**What was done:**
- #340 Add the Greek translation by Voutsas Theocharis
- Improved Portuguese (Brazil) translation by Everton Massen Goncalves
- #344 Add support of TLS transport for User and Provider settings
- #356 Add ability to configure outbound_proxy in Provider settings
- #365 Extend VoIP Provider settings. Added "Inbound", "Outbound" and "Without registration" options
- #341 Add weak password validation for web and SSH accounts
- #343 Add support G.729 audio codec
- #155 Add additional transfer options to Incoming and Forwarding route rules "Hang up","Busy", "Redirect according to DID number"
- #310 Fix CDR logging for voicemail messages
- #339 Optimize boot performance. Put SWAP initialization into background process. 
- #307 Fix the error of recording the call history. When the call arrived at the queue and the subscriber tried attended transfer the call to another queue, the conversation was not recorded
- #346 Add username validation to prevent using special symbols there
- #351 Hide the SSH menu item if browser not support the function
- #358 Fix REST API issues if there is no the Internet connection
- #352 Improve the Yandex Cloud provisioning function. Auto setup Password, IP, Gateway, DNS and Firewall settings.
- #308 Improve label's translation on Outbound rules page
- #325 Fix the list of modules table view
- #333 Fix conversation issues from WAV to MP3
- #348 Update SSL CA certs and certificate validation issues
- #314 Fix the frequency of "weak SSH password" notification 
- #322 Improve firmware update process add extra notification to the serial console
- Add orphaned sound files to German system language
- #294 Extend timeout waiting the Redis service start
- Fix Fil2Ban rules to prevent web password attack
- Add notification about the default password
- #147 Add internet connection check before filling registration form on Marketplace page
- #274 Improve coupon activation status notification
- #355 Improve email notification when free space on storage disk is running out
- #319 Add busy code return if extension is busy
- #350 Change module upgrade URL to releases.mikopbx.com
- Improve security in default firewall settings for 0.0.0.0 network
- #354,#361 Fix module turn on/off and update proces
- #362 Optimize Iptables settings
- #340 Filter unused sound files for all available languages
- #374 Fix disk mount issues
- Add extra outbound context customization feature
- #317 Fix CDR records when redirecting to mobile
- Block possibility to login under www user
- #312 Change the night switch context priority
- #357 Add IVR missed call notification on email
- #316 Fix CDR errors when call forwarded to a queue
- Fix Asterisk issues if externalHostName option does not resolve by DNS 
- #365 Remove defaultuser option on Provider page
- #349 Optimize the first start process
- #29 Fix dynamic agent priority in a queue, fix number template to use one-digit numbers 

<hr /><em>This discussion was created from the release <a href='https://github.com/mikopbx/Core/releases/tag/2022.2.102'>MikoPBX 2022.2.102</a>.</em>

**Comment by AlekZone** [comment]
inbound rules broken after remove demo?!
[123](https://disk.yandex.ru/i/lvQGE6-hJG4tTA)

**Reply by boffart** [reply]
> inbound rules broken after remove demo?! [123](https://disk.yandex.ru/i/lvQGE6-hJG4tTA)

I don't understand you. What is not working correctly? Describe the procedure for repeating the situation

**Reply by AlekZone** [reply]
- intall new from iso
- delete demo users
- add new user
but add incoming route can't select no queue no users

**Reply by boffart** [reply]
Try restarting the PBX

**Reply by AlekZone** [reply]
restart did not help)
